### PR TITLE
nvuApp: admin now can view all questionnaires

### PR DESCRIPTION
**Pull request by qnislavnachev**
added a function to admin panel to view all
questionnaries that are created

Related #9

### CODE REVIEW

**Comment by qnislavnachev** [comment]
PR need a review

File: src/main/resources/com/clouway/nvuapp/adapter/http/servlet/adminHome.html [hunk 1]
```diff
@@ -32,5 +32,10 @@
         <input class="buttonlink" type="submit" value="Редактирай въпросник"/>
     </form>
 </div>
+<div>
```
Comment (mgenov) [inline src/main/resources/com/clouway/nvuapp/adapter/http/servlet/adminHome.html]: Simple link could do the job.

File: src/main/java/com/clouway/nvuapp/adapter/http/controllers/ViewQuestionnaireHandler.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+public class ViewQuestionnaireHandler implements SecuredHandler {
+    private QuestionnaireRepository repository;
+
+    public ViewQuestionnaireHandler(QuestionnaireRepository repository) {
+        this.repository = repository;
+    }
+
+    @Override
+    public Response handle(Request request, Tutor tutor) {
+        Integer id = Integer.valueOf(request.param("id"));
```
Comment (mgenov) [inline src/main/java/com/clouway/nvuapp/adapter/http/controllers/ViewQuestionnaireHandler.java]: Someone could pass unknown id and this case is not handled here and by tests.

**Comment by qnislavnachev** [comment]
view handler now handle the case if the id is unknown

File: src/main/java/com/clouway/nvuapp/adapter/http/controllers/ViewQuestionnaireHandler.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        if (repository.getQuestionnaire(id) == null) {
```
Comment (mgenov) [inline src/main/java/com/clouway/nvuapp/adapter/http/controllers/ViewQuestionnaireHandler.java]: This code will execute same SQL query twice to the server, which doesn't sound good.

**Comment by qnislavnachev** [comment]
in view handler no longer  SQL query is executed twice

**Comment by mgenov** [comment]
PR has 2 commits and I'm not able to merge it.

**Comment by qnislavnachev** [comment]
no longer pull request has 2 commits now they are squashed

**Pull request merged**